src/js/themes/minimal.js

```diff
@@ -0,0 +1,230 @@
+import { window, document } from '../utils/server-safe-globals.js';
+import { MediaThemeElement } from '../media-theme-element.js';
+
+const html = (raw, ...keys) => String.raw({ raw }, ...keys);
+
+const template = document.createElement('template');
+template.innerHTML = html`
+  <style>
+    media-controller {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
```

Comment:
the time range and volume range has a border-radius in design.
there's a CSS var for that `--media-range-track-border-radius`